src/app/features/profile/index.js

```
@@ -0,0 +1,18 @@
+
+const profileSlice = createSlice({
+  name: "openletter",
+
+  initialState: {
+    isLoading: false,
+    error: null,
+  },
+
+  reducers: {},
+
+  extraReducers: (builder) => {},
```

ESLint

[error] 'builder' is defined but never used.

(no-unused-vars)